.github/workflows/documentationjs.yml

```diff
@@ -0,0 +1,21 @@
+name: Deploy documentation.js on GitHub pages
```

Comment:
Sorry I forgot to mention in the issue that we should not add a documentationjs workflow if there is already a typedoc workflow